core/domain/suggestion_registry_test.py

```diff
@@ -382,25 +382,6 @@ def test_validate_score_type_content(self):
             'Expected the first part of score_category to be content'):
             suggestion.validate()
 
-    def test_validate_score_sub_type(self):
```

Comment:
Should we have a test for the sub_type to be a random category?

Reply:
I can add that...